pkg/terminal/auth.go

```diff
@@ -7,10 +7,10 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// checkUserPermissions checks if the terminal proxy is supported for a given user.
-// Returns true if we're willing to proxy the user's token, false otherwise. We don't
-// want to proxy highly privileged tokens to avoid privilege escalation issues.
-func (p *Proxy) checkUserPermissions(token string) (bool, error) {
+// isClusterAdmin does a subject access review to see if the user can create pods in openshift-operators
```

Comment:
The namespace in the comment doesn't match the code.

Reply:
Darn, that was the one spot that I forgot to change! Should I change it quickly or will it remove the LGTM status?